cms/djangoapps/contentstore/views/tests/test_preview.py

```diff
@@ -213,11 +213,7 @@ def test_expected_services_exist(self, expected_service):
         Tests that the 'user' and 'i18n' services are provided by the Studio runtime.
         """
         block = BlockFactory(category="pure", parent=self.course)
-        _prepare_runtime_for_preview(
-            self.request,
-            block,
-            self.field_data,
```

Comment:
Same here - there are a couple initializations of `self.field_data` in `setUp()` that now seem unused.

Reply:
Thanks for catching this. I removed these in 1d42e77.